### DISCUSSION TITLE
Welcome to Brightness-Sync Discussions!

### DISCUSSION THREAD

**Discussion by OCJvanDijk** [OCJvanDijk/Brightness-Sync#35] [General]
## 👋 Welcome!
Until now people have been using the Issues page for general comments, questions and even thank you's (much appreciated!).
Github has a Discussions feature now so why not just enable it? 🙂

**Comment by Maqsim** [comment]
@OCJvanDijk Hey dude. This is such amazing app. I have MacBook M1 Max and LG UltraFine 22". Works like a charm. I wanted to buy you a coffee, but I don't have a PayPal account (I am from Ukraine).

**Reply by OCJvanDijk** [reply]
Thanks! Don’t worry about the coffee, I’m glad just to hear you’re enjoying the app :)

Since you’re on M1, I’d be interested to hear if you have any of the problems reported [here](https://github.com/OCJvanDijk/Brightness-Sync/discussions/37)? I’m still on Intel and still haven’t tested it for myself…
